### ISSUE TITLE
Galant Schemata

### ISSUE THREAD

**Issue by bcmoseley** [openmusictheory/openmusictheory.github.io#8]
@kshaffer This seems to be something you're interested in supplementing, right? I'm going to mark it for the relaunch, but if you think it's better to wait, I don't think it's absolutely necessary.

What do you think needs to be added?

**Comment by kshaffer** [comment]
Yes, I'll want to take a look at this this before September, at the latest, since we'll be using it again in class. I'll come back and note what I think needs to be added here.

**Comment by bcmoseley** [comment]
Note that a while back I did create [Galant Schemata: Opens and Closes](http://www.openmusictheory.com/schemataOpensAndCloses.html) and [Galant Schemata: Continuation Patterns](http://www.openmusictheory.com/schemataContinuationPatterns.html) if you want to use those.

**Comment by bcmoseley** [comment]
Because we're running out of time, I'm going to remove this from the August relaunch milestone. Hopefully we'll be able to get this done during the fall.
